compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -50,9 +49,16 @@ case class State(objectIndex: Int,
                  imports: Map[Int, List[tpd.Import]],
                  context: Context)
 
-/** Main REPL instance, orchestrating input, compilation and presentation */
+/** Main REPL instance, orchestrating input, compilation and presentation.
+ *
+ * @param out where regular output (eg. "define trait Foo") is printed
+ * @param errorReporter if set, this is reporter is passed to the parser and typer. Otherwise,
+ *        the warnings and errors are printed to `out` as well. Note, however, that REPL errors
+ *        (eg. ":load" cannot find the file) are not reported with this reporter
```

Comment:
Any particular reason for this? Seems like from a user point of view it would make sense to have those handled by the reporter too.

Reply:
nah, not really a reason. I just wasn't sure if it's required (see PR description). I'll add it :+1: